src/App.js

```diff
@@ -4,9 +4,19 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
 
   render() {
+    const movies = moviesFromServer.filter(
+      movie => !this.state.query
```

Comment:
this is not very explicit, you better write this before filtering and do filtering according to this value